src/system/startup.c

```diff
@@ -40,6 +40,8 @@ __attribute__((constructor(101))) static void pros_init(void) {
 
 	vdml_initialize();
 
+	graphical_context_daemon_initialize();
```

Comment:
This is the second graphical daemon; Is there a way to combine them? I assume using these APIs isn't going to be common because, well, it's the third graphical API, and you're adding it after people who use it have experience with lvgl. In that case, it isn't the best to run this by default, as it makes more work happen for everyone else, including those who never intend to use these APIs.

Reply:
Given that our plans are to decouple LVGL from the Kernel in the near future, these are seperate intentionally.

Reply:
Yeah I get what you mean, this definently needs some rfc in terms of creating a system where users can easily disable or enable the screen APIs. For now though, getting this feature out for testing and feedback takes priority.